### PR TITLE
Add diagramms (Edit task, User registration) to documentation

### PR DESCRIPTION
**Pull request by ashatna**
## JIRA

* [Main JIRA ticket](https://ssu-jira.softserveinc.com/browse/LEAR-13)

## Code reviewers

- [ ] @ashatna
- [ ] @ChristinaPastushchak
- [ ] @dotiurin
- [ ] @iryn34
- [ ] @tarassito
- [ ] @VasilHamulyak
- [ ] @VitalyiKhlivnyi
- [ ] @YurijH

### Second Level Review

- [ ] @ushevtsiv
- [ ] @lhalam
- [ ] @ssOleg

## Summary of issue

ToDo

## Summary of change

ToDo

## Testing approach

ToDo

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

**Review by VasilHamulyak — DISMISSED** [review]

**Review by iryn34 — DISMISSED** [review]

File: documentation/sequence diagrams/authentication/User Registration Diagramm.txt [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    Models->+DataBase:writes User object into the DB
+    DataBase->-Models: Set id
+    Models->Authentication/views: User object
+    deactivate Models
+    Authentication/views->+User email: send_mail(activation)
+    Authentication/views-->UI component: response (status 201)
+    deactivate UI component
+    deactivate Authentication/views
+    User->User email: click link
+    User email->+Authentication/views: Get request
```
Comment (tarassito) [inline documentation/sequence diagrams/authentication/User Registration Diagramm.txt]: There is no Token object and return RESPONSE_498_INVALID_TOKEN

File: documentation/sequence diagrams/authentication/User Registration Diagramm.txt [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    deactivate User email
+    Authentication/views->+Models: User update
+    Authentication/views->Models: CustomerProfile create
+    Models->+DataBase: writes CustomProfile object into the DB
+    DataBase->-Models: Set id
+    Models->Authentication/views: CustomProfile object
```
Comment (tarassito) [inline documentation/sequence diagrams/authentication/User Registration Diagramm.txt]: there is no return RESPONSE_200_ACTIVATED

File: documentation/sequence diagrams/authentication/User Registration Diagramm.txt [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+participant UI component
+participant Authentication/views
+participant Validators
+participant Models
+participant User email
+participant DataBase
+
+User->+UI component: input data
+User->UI component: click 'register' button
+UI component->+Authentication/views: registration(post request)
```
Comment (tarassito) [inline documentation/sequence diagrams/authentication/User Registration Diagramm.txt]: there is no return RESPONSE_400_EMPTY_JSON , RESPONSE_400_EXISTED_EMAIL

File: documentation/sequence diagrams/task/Edit Task Diagramm.txt [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+deactivate DataBase
+deactivate Task Model
+Task View->+Task Model: updates title, description, status
+Task Model->+DataBase: set changes into DB
+DataBase->Task Model: return task id
+deactivate DataBase
+deactivate Task Model
+Task View->EditTask Dialog: response_200_updated
+deactivate Task View
+deactivate EditTask Dialog
```
Comment (tarassito) [inline documentation/sequence diagrams/task/Edit Task Diagramm.txt]: there are no Responses 400

File: documentation/sequence diagrams/authentication/User Registration Diagramm.txt (hunk 3 above)
Comment (ashatna) [reply documentation/sequence diagrams/authentication/User Registration Diagramm.txt]: done

**Review by tarassito — DISMISSED** [review]

**Review by VitalyiKhlivnyi — APPROVED** [review]

**Pull request merged**